### PR TITLE
Release 2.0-beta8 is the current devel release

### PR DESCRIPTION
**Pull request by nskaggs**
This includes an update to the temp-release-notes.

### CODE REVIEW

File: src/en/temp-release-notes.md [hunk 1]
```diff
@@ -339,15 +230,15 @@ and credentials that involves less editing of files and makes Juju work
 out of the box with major public clouds like AWS, Azure, Joyent,
 Rackspace, Google, Cloudsigma.
 
-Firstly, there is no more environments.yaml file to edit. Clouds and
+There is no environments.yaml file to edit. Instead, dlouds and
```
Comment (pmatulis) [inline src/en/temp-release-notes.md]: clouds

File: src/en/temp-release-notes.md [hunk 2]
```diff
@@ -1553,16 +1444,17 @@ Juju now supports Windows 10 Enterprise and Windows Server 2016
...
     juju deploy
     Juju upgrade-charm
 
+For more information on the new support for channels in the Charm Store
+and how they work, please see our
+[documentation](https://jujucharms.com/docs/devel/authors-charm-store
+#entities-explained) on the subject.
+
 
 ### Keystone 3 support in Openstack.
```
Comment (pmatulis) [inline src/en/temp-release-notes.md]: OpenStack

File: src/en/temp-release-notes.md [hunk 3]
```diff
@@ -1572,8 +1464,8 @@ Juju now supports Openstack with Keystone Identity provider V3. Keystone
 user/password too) juju will use V3 authentication by default. In other
 cases where only user and password is present, it will query Openstack
```
Comment (pmatulis) [inline src/en/temp-release-notes.md]: OpenStack

**Comment by evilnick** [comment]
Thanks :cake:

**Pull request merged**